docs/subquery_network/node_operators/maximise-apy.md

```diff
@@ -0,0 +1,171 @@
+
+We strongly suggest that you log in at the start of each new Era (as close as possible to the start - set a calendar reminder), to resolve any warnings or notifications, as well complete the following key tasks.
+
+### Weekly tasks - Claim rewards
+
+If you don't frequently claim your rewards (e.g. once a week), delegators can see this and may decide against delegating to support you. Make sure you claim your rewards frequently (e.g. once a week), and ideally as soon as they become available after the end of the past Era.
+
+Alternatively, ensure enough ETH balance is in your Controller Account so that rewards can be collected automatically. This will result in an Era Reward Collection status of “Frequent” on the [All Node Operators](https://app.subquery.network/indexer/all-indexers) dashboard meaning that your Delegators can also receive their rewards.
+
+### Weekly tasks - Check for new project versions
+
+Project creators might deploy new versions of their project codebases each time. We strongly suggest following their release notes and updating your project to match the recommended versions. In some cases you wont need to reindex, just update the project version.
+
+### Weekly tasks - Update SQT allocation
+
+- Avoid having tokens allocated to outdated deployment versions. An “Outdated Allocation Projects” pop up will appear if this occurs. This means that you should adjust your allocation to the latest version (and update to the new version).
+- Avoid having “Unallocated Stake”. To maximise rewards, ensure all tokens are allocated. An “Unallocated Stake” pop up will appear if this occurs. 
+
+### Weekly tasks - Check Project health
+- Ensure your project is online. Appending “/healthy” to the end of your http URL should return:
+```
+{
+  "autoReduceAllocation": true,
+  "controller": "",
+  "coordinatorVersion": "v2.2.3",
+  "indexer": "",
+  "os": "Debian GNU/Linux 10 4-CPU",
+  "proxyVersion": "v2.4.2",
+  "uptime": 8066
+}
+```
+- Ensure you have at least one flexplan enabled for each project.
+- Ensure projects are able to fetch metadata from deployment’s Query Service. An error will appear in the status column if there is an issue.
+
+## Selecting what projects to run.
+
+As a node operator, three types of projects can be run: SubQuery data indexers, Subgraphs, or RPC endpoints. See https://app.subquery.network/explorer/home for a complete list. 
+
+Rewards are generated by each project based on the amount of boost allocated to a project and then shared proportionally between the number of operators (for that project).
+
+What to look out for? Projects with:
+- high boost and therefore high stake rewards
+- high query numbers
+- low competition
+- low project complexity
+
+Project database size is a good indicator of cost
+
+## Node Operator Rewards FAQs
+
+### Which projects should I run to generate the highest rewards?
```

Comment:
This is a duplicate of the above heading? Can we combine?

Reply:
agree. done.